src/compiler/data.ts

```diff
@@ -175,9 +182,10 @@ export namespace source {
    * @return An array that might contain a filter transform for filtering null value based on filterNul config
    */
   export function nullFilterTransform(model: Model) {
-    const filterNull = model.config('filterNull');
+    const filterNull = model.config().filterNull;
     const filteredFields = keys(model.reduce(function(aggregator, fieldDef: FieldDef) {
-      if (fieldDef.field && fieldDef.field !== '*' && filterNull[fieldDef.type]) {
+      if (filterNull === true ||
+        filterNull !== false && fieldDef.field && fieldDef.field !== '*' && DEFAULT_NULL_FILTERS[fieldDef.type]) {
```

Comment:
Hmm, I wonder whether 'auto' should also work. That's why I wrote it the way it is. But we should be consistent.